src/components/Counter/Counter.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import './Counter.scss';
+
+export class Counter extends React.Component {
+  state = {
+    count: 0,
+    cklicks: 0,
+  }
+
+  addOne = () => {
+    this.setState(prevState => ({
+      count: prevState.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState(prevState => ({
+      count: prevState.count + 100,
+    }));
+  };
+
+  increase = () => {
+    if (this.state.cklicks < 5 && this.state.cklicks !== 0) {
+      this.setState(prevState => ({
+        cklicks: prevState.cklicks + 1,
+      }));
+      this.addOne();
+    } else {
+      this.setState({
+        cklicks: 1,
+      });
+      this.addOne();
+      this.add100();
+    }
+  }
```

Comment:
```suggestion
  increase = () => {
    this.setState(({ count }) => {
      this.addOne();

      if (count % 5 === 0 && count !== 0) {
        this.add100();
      }
    });
  };
```
Не совсем понимаю, зачем тебе нужна переменная cklicks. Можно ж сам count увеличивать

Reply:
Illia is right